### PR TITLE
Re-enable a11y validation and fix issues

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: README.md [hunk 1]
```diff
@@ -43,3 +43,6 @@
   ```
   npm run test
   ```
+
+## Important
+Since travis is failing on executing a11y tests due to out of date suid you need to execute `npm run a11y` locally before sending any pr to verify we're following accessibility standars.
```
Comment (matuzalemsteles) [inline README.md:48]: What do you think of adding `npm run a11y` in `lint-staged`?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "a11y": "metal-a11y --packages ./packages -c demos/index.html",
+    "a11y": "metal-a11y --packages ./packages -c demos/a11y.html",
```
Comment (matuzalemsteles) [inline package.json:4]: hey @carloslancha, why two files in the `index` and `a11y` demos?

Comment (carloslancha) [reply package.json:4]: `index` is for demo purposes, `ally` only for accessiblity. There're times when we are not following accessiblity due to the config we pass to the components (like in ClayList, where we're passing the same items to all instances in behalf of simplicity) but that doesn't mean component itself is not following a11y. That's where `ally.html` shows up.

File: README.md (hunk 1 above)
Comment (carloslancha) [reply README.md:48]: I think `lint-staged` is fine for linting and format stuff, but if we add `ally` we force each commit yo be accessible, which I think is not a good idea while developing.

Comment (matuzalemsteles) [reply README.md:48]: Yeah, that might have gone bad, I had not thought about that part 😅.

File: package.json (hunk 2 above)
Comment (matuzalemsteles) [reply package.json:4]: This is good. The bad thing is that we will have another file for maintenance.

Comment (carloslancha) [reply package.json:4]: When we have all Clay+claycss.com+components fully linked in the project demos probably will be useless. Since then...

**Pull request merged**